src/routes/safe/components/Transactions/TxList/styled.tsx

```diff
@@ -82,32 +78,39 @@ export const SubTitle = styled(Text)`
 `
 
 export const StyledTransactions = styled.div`
-  background-color: ${({ theme }) => theme.colors.white};
-  border-radius: 8px;
-  box-shadow: #00000026 0 4px 12px 0;
   overflow: hidden;
   width: 100%;
 
+  display: flex;
+  flex-direction: column;
+  row-gap: 6px;
+
   & > .MuiAccordion-root {
+    border: 2px solid ${grey400};
+    border-radius: 8px;
+
     &:first-child {
-      border-top: none;
+      border: 2px solid ${grey400};
     }
 
-    &:last-child {
-      border-bottom: none;
+    & .MuiAccordionSummary-root.Mui-expanded,
+    & .MuiAccordionSummary-root:hover {
+      background-color: ${primary200};
+    }
+
+    &.Mui-expanded {
+      border: 2px solid ${primary300};
     }
   }
 `
 
-export const GroupedTransactionsCard = styled(StyledTransactions)`
+export const GroupedTransactionsCard = styled(StyledTransactions)<{ expanded?: boolean }>`
   transition: all 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
-  background-color: transparent;
-  border-radius: 0;
-  box-shadow: none;
+  background-color: ${({ theme }) => theme.colors.white}};
 
-  &:not(:last-child) {
-    border-bottom: 2px solid ${({ theme }) => theme.colors.separator};
-  }
+  border: 2px solid ${({ expanded }) => (expanded ? `${primary300}` : `${grey400}`)};
+  width: calc(100% - 4px);
```

Comment:
You can achieve the same by using `box-sizing: border-box` so that the border is included in the box dimensions.

Reply:
just applied it. works like a charm